docs/tutorials/contributing/strategy/writing_the_new_strategy.rst

```diff
@@ -7,7 +7,7 @@ Identify a new strategy
 The library has a large number of strategies::
 
     >>> import axelrod as axl
-    >>> len(axl.strategies)
+    >>> len(axl.strategies) # doctest: +SKIP
     103
 
 If you're not sure if you have a strategy that has already been implemented
```

Comment:
Well at least the diagrams will show how many strategies their are. We could also say that "the library has at least X many strategies" and periodically update that.

Technically the library can generate infinitely many strategies.